core/templates/services/platform-feature.service.spec.ts

```diff
@@ -46,10 +46,8 @@ describe('PlatformFeatureService', () => {
 
   let apiSpy: jasmine.Spy;
 
-  // TODO(#9154): Remove the following resetting code when migration is
```

Comment:
Why remove this TODO?

Reply:
#9154 has been closed so there's no need to force the member to be static, but I still think static members can be a good guard to prevent this service to be multi-instantiated, so I only removed the TODO comments (per [this thread in #10185](https://github.com/oppia/oppia/pull/10185#discussion_r473632233) )

Reply:
Suggest adding comments in relevant places that these should kept as static.

Reply:
Done, added comments for it in `platform-feature.service.ts`